### DISCUSSION TITLE
Stack on an android

### DISCUSSION THREAD

**Discussion by kvi77** [immich-app/immich#7315] [Q&A]
There is a problem with creating a stack on an android application. I select several images, click on the stack, the images are not stacked.
Disassembling the stack doesn't work either. It started a couple of updates ago, I can't say exactly when.
Now, the server and android app version is 1.95.1

**Comment by alextran1502** [comment]
confirm it is a bug

**Reply by kvi77** [reply]
Here is a short video showing this error

https://github.com/immich-app/immich/assets/138368171/f5eb00b4-61ed-465c-9df1-95c38b00394c
